crates/sui-framework/packages/move-stdlib/sources/ascii.move

```diff
@@ -91,15 +91,15 @@ module std::ascii {
     public fun insert(s: &mut String, at: u64, o: String) {
         assert!(at <= s.length(), EInvalidIndex);
         let l = s.length();
-        let mut front = s.sub_string(0, at);
-        let end = s.sub_string(at, l);
+        let mut front = s.substring(0, at);
+        let end = s.substring(at, l);
         front.append(o);
         front.append(end);
         *s = front;
     }
 
     /// Copy the slice of the `string` from `i` to `j` into a new `String`.
-    public fun sub_string(string: &String, mut i: u64, j: u64): String {
+    public fun substring(string: &String, mut i: u64, j: u64): String {
```

Comment:
I don't understand how we're able to land a change that changes the name of a public function.